pkg/log/log.go

```diff
@@ -65,10 +71,103 @@ func (l *OsqueryLogAdapter) Write(p []byte) (int, error) {
 		return len(p), nil
 	}
 
+	if bytes.Contains(p, []byte("Refusing to kill non-osqueryd process")) {
+		l.logInfoAboutUnrecognizedProcessLockingPidfile(p)
+	}
+
 	msg := strings.TrimSpace(string(p))
 	caller := extractOsqueryCaller(msg)
 	if err := lf(l.logger).Log(append(l.extraKeyVals, "msg", msg, "caller", caller)...); err != nil {
 		return 0, err
 	}
 	return len(p), nil
 }
+
+// Occasionally, launcher will fail to start osquery -- in this case, osquery fails
```

Comment:
The comment is good, but it's in the place of a godoc function comment. So it should either be formatted as such, or moved.

Reply:
Addressed, thank you!